contracts/ito.sol

```diff
@@ -154,25 +148,10 @@ contract HappyTokenPool {
             pool.exchanged_tokens.push(0); 
         }
 
-        // Make sure each ratio is co-prime to prevent overflow
-        for (uint256 i = 0; i < _ratios.length; i+= 2) {
-            uint256 divA = SafeMath.div(_ratios[i], _ratios[i+1]);      // Non-zero checked by SafteMath.div
-            uint256 divB = SafeMath.div(_ratios[i+1], _ratios[i]);      // Non-zero checked by SafteMath.div
-            
-            if (_ratios[i] == 1) {
-                require(divB == _ratios[i+1]);
-            } else if (_ratios[i+1] == 1) {
-                require(divA == _ratios[i]);
-            } else {
-                // if a and b are co-prime, then a / b * b != a and b / a * a != b
-                require(divA * _ratios[i+1] != _ratios[i]);
-                require(divB * _ratios[i] != _ratios[i+1]);
-            }
-        }
         pool.ratios = _ratios;                                          // 256 * k
         IERC20(_token_addr).safeTransferFrom(msg.sender, address(this), _total_tokens);
 
-        emit FillSuccess(_total_tokens, _id, msg.sender, block.timestamp, _token_addr, message);
+        emit FillSuccess(msg.sender, _id, _total_tokens, block.timestamp, _token_addr, message);
```

Comment:
why? do indexed variables need to be placed as the first two?

Reply:
I think it is natural to have the `indexed` variable as the first variable. If you do not like it, I think we can change it back.

Note: `_id` is not indexed here, `_id` is `random` in the context(to the pool owner), so it does not make much sense to have an indexed `_id`.

Reply:
thanks I just want to make sure if I am missing some standards. just keep it there then